src/services/docGenService.ts

```diff
@@ -107,6 +110,7 @@ export class DocGenService {
                 adapter,
                 message,
                 columns,
+                1,
```

Comment:
this should be columnCount?

Reply:
ColumnIndex right?

Reply:
actually no. It should be completedColumns count and is applicable only for doc gen